libcxx/src/support/runtime/exception_pointer_cxxabi.ipp

```diff
@@ -34,6 +35,29 @@ exception_ptr& exception_ptr::operator=(const exception_ptr& other) noexcept
     return *this;
 }
 
+#  if _LIBCPP_AVAILABILITY_HAS_INIT_PRIMARY_EXCEPTION
```

Comment:
Here, just remove the `#if` guard and it'll make sure we define these symbols whenever we build the dylib, which is what we want.

The availability macros are meant to guard whether the code in the headers should rely on these definitions in the dylib to support back-deployment use cases. But we always want to define those symbols in the dylib.

Reply:
Done.